CI/travis.osx.after_success.sh

```diff
@@ -34,19 +34,21 @@ if [[ "${MUDLET_VERSION_BUILD}" == -ptb* ]]; then
   public_test_build="true"
 fi
 
-# we deploy only certain builds
+# deploy is set to "deploy" on all Github actions. Is there a case where we would run this but not deploy?
```

Comment:
Commenting here for a discussion. This entire block is if-cased around $DEPLOY = "deploy", but it's never not deploy, so should we remove this?

Reply:
I think the idea is that you can also run this script outside of the CI environment as well and for consistency with the Linux/Windows equivalents